pkg/util/node/node.go

```diff
@@ -103,46 +96,6 @@ func GetNodeHostIPs(node *v1.Node) ([]net.IP, error) {
 	return nodeIPs, nil
 }
 
-// GetNodeHostIP returns the provided node's "primary" IP; see GetNodeHostIPs for more details
-func GetNodeHostIP(node *v1.Node) (net.IP, error) {
```

Comment:
https://cs.k8s.io/?q=GetNodeHostIP&i=nope&files=&excludeFiles=&repos=

This is used in other repo like https://github.com/kubernetes/autoscaler/blob/cc6ddad1539e9a57ada158d945af3d3cbf2a037e/vertical-pod-autoscaler/e2e/vendor/k8s.io/kubernetes/pkg/kubelet/kubelet_getters.go#L285.

Reply:
autoscaler isn't using this function, it's just vendoring all of k/k, so that it can use some other functions.